base/nvti.c

```diff
@@ -329,50 +335,41 @@ nvti_refs (const nvti_t *n, const gchar *type)
       if (type && strcmp (ref->type, type) != 0)
         continue;
 
-      if (refs)
-        refs2 = g_strdup_printf ("%s,%s:%s", refs, ref->type, ref->ref_id);
-      else
-        refs2 = g_strdup_printf ("%s:%s", ref->type, ref->ref_id);
-      g_free (refs);
-      refs = refs2;
-    }
-
-  return (refs);
-}
-
-/**
- * @brief Get the xref's.
- *
- * @param n The NVT Info structure of which the name should
- *          be returned.
- *
- * @return The xref string. Don't free this.
- */
-gchar *
-nvti_xref (const nvti_t *n)
-{
-  gchar *xrefs, *xrefs2;
-  vtref_t * ref;
-  guint i;
-
-  if (! n) return (NULL);
+      exclude_item = exclude_split;
+      exclude = 0;
+      while (*exclude_item)
+        {
+          if (strcmp (g_strstrip (*exclude_item), ref->type) == 0)
+            {
+              exclude = 1;
+              break; 
+            }
+          exclude_item ++;
+        }
 
-  for (i = 0; i < g_slist_length (n->refs); i ++)
-    {
-      ref = g_slist_nth_data (n->refs, i);
-      if ((strcmp (ref->type, "cve") != 0) &&
-          (strcmp (ref->type, "bid") != 0))
-        { // if it is neither cve nor bid, then it is some xref
-          if (xrefs)
-            xrefs2 = g_strdup_printf ("%s,%s:%s", xrefs, ref->type, ref->ref_id);
+      if (! exclude)
```

Comment:
Fixed.

Reply:
I don't see this fix.  And I see some other parts of this PR that add the space after the !.
The rest of the code does not do this (since Hani's formatting change).
We should really have a formatting check for the PRs.

Reply:
Thought you meant a trailing space which I fixed. Now really fixed.